### DISCUSSION TITLE
How get current record id in Relation Manager

### DISCUSSION THREAD

**Discussion by HomaEEE** [filamentphp/filament#3959] [Help] [answered]
in tab I have 2 relation tables, in second form I need get Parent Id of record for select rooms only this record
<img width="1109" alt="Снимок экрана 2022-09-11 в 19 11 03" src="https://user-images.githubusercontent.com/1429343/189537794-10b80f85-1805-4917-838e-ebae39f3e76e.png">

```
 return $form
            ->schema([
                TextInput::make('table_id')->label(__('Table_id')),
                Select::make('room_id')->options(Room::all()->pluck('room', 'id')),
                TextInput::make('place_id') //id of current place record
            ]);
```

Structure: Place -> Room -> Seets

**Comment by danharrin** [comment, marked as answer]
You don't need that input at all, it is filled by Filament automatically. You can remove that place_id field

If you really need to access it, try `default(fn ($livewire) => $livewire->ownerRecord->id)`
